charts/harvester-load-balancer/values.yaml

```diff
@@ -30,7 +30,7 @@ securityContext: {}
 resources:
   limits:
     cpu: 100m
-    memory: 128Mi
+    memory: 256Mi
```

Comment:
I suggest filing a PR against the `master` branch first if there are more updates to the chart manifest than trivial version updates. PRs against the `release` branch should better contain only version updates and commits cherry-picked from the `master` branch. Thank you.

Reply:
Yeah, it is reasonable. I will change the memory limits on anothter PR later. thanks.